README.md

```diff
@@ -48,7 +48,7 @@ That's it.
 This plugin is compatible with:
 
 - PHP **5.4+**, **7.x**, and **8.x** (Support for PHP v8 is available since [`v0.7.0`][v0.7])
-- [Composer][composer] **1.x** and **2.x** (Support for Composer v2 is available since [`v0.7.0`][v0.7])
+- [Composer][composer] **2.2+** (Support for Composer v2 is available since [v0.7.0][v0.7]; support for Composer < 2.2 was dropped in [v1.1.0][v1.1])
```

Comment:
```suggestion
- [Composer][composer] **2.2+** (Support for Composer v2 is available since [`v0.7.0`][v0.7]; support for Composer < 2.2 was dropped in [`v1.1.0`][v1.1])
```

Reply:
:+1: